src/Dapr.Client/StateQueryResponse.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+
+namespace Dapr.Client
+{
+    /// <summary>
+    /// Represents the response from a state query.
+    /// </summary>
+    public class StateQueryResponse<TValue>
+    {
+        /// <summary>
+        /// The results of the query.
+        /// </summary>
+        public IReadOnlyList<StateQueryItem<TValue>> Results { get; set; }
+
+        /// <summary>
+        /// The pagination token to continue the query.
+        /// </summary>
+        public string Token { get; set; }
```

Comment:
The API in this PR defines no way to send this back as part of a request. How does this work?

Reply:
You send it back in the query itself actually. It's part of the `pagination` section. I'm not sure how to relay that in the SDK without just linking to the docs. Or is this where we should put in a simple example?

https://docs.dapr.io/developing-applications/building-blocks/state-management/howto-state-query-api/#example-3

Reply:
Docs are fine, I it's a little unfortunate building queries as strings **and** building them with interpolation.